node/request.js

```diff
@@ -198,6 +198,8 @@ TChannelRequest.prototype.send = function send(arg1, arg2, arg3, callback) {
 TChannelRequest.prototype.resend = function resend() {
     var self = this;
 
+    if (self.channel.destroyed) return;
```

Comment:
Talked to @Raynos, I will update TChannelConnectionBase.prototype.resetAll to handle this scenario. I will do it in a separate PR.